pkg/Microsoft.NETFramework.Compatibility/Microsoft.NETFramework.Compatibility.pkgproj

```diff
@@ -20,40 +20,51 @@
+    <PrereleaseLibraryPackage Include="System.DirectoryServices.AccountManagement" />
+    <PrereleaseLibraryPackage Include="System.DirectoryServices.Protocols" />
+    <PrereleaseLibraryPackage Include="System.IO.FileSystem.AccessControl" />
+    <PrereleaseLibraryPackage Include="System.IO.Packaging" />
+    <PrereleaseLibraryPackage Include="System.IO.Pipes.AccessControl" />
+    <PrereleaseLibraryPackage Include="System.IO.Ports" />
+    <PrereleaseLibraryPackage Include="System.Management" />
+    <PrereleaseLibraryPackage Include="System.Security.AccessControl" />
+    <PrereleaseLibraryPackage Include="System.Security.Cryptography.Pkcs" />
+    <PrereleaseLibraryPackage Include="System.Security.Cryptography.ProtectedData" />
+    <PrereleaseLibraryPackage Include="System.Security.Cryptography.Xml" />
+    <PrereleaseLibraryPackage Include="System.Security.Permissions" />
+    <PrereleaseLibraryPackage Include="System.Security.Principal.Windows" />
+    <PrereleaseLibraryPackage Include="System.ServiceModel.Syndication" />
+    <PrereleaseLibraryPackage Include="System.ServiceProcess.ServiceController" />
+    <PrereleaseLibraryPackage Include="System.Text.Encoding.CodePages" />
+    <PrereleaseLibraryPackage Include="System.Threading.AccessControl" />
+
+    <!-- Stable packages shipped already for netcoreapp2.0 -->
     <LibraryPackage Include="System.Security.Cryptography.Cng" />
-    <LibraryPackage Include="System.Security.Cryptography.Pkcs" />
-    <LibraryPackage Include="System.Security.Cryptography.ProtectedData" />
-    <LibraryPackage Include="System.Security.Cryptography.Xml" />
-    <LibraryPackage Include="System.Security.Permissions" />
-    <LibraryPackage Include="System.Security.Principal.Windows" />
-    <LibraryPackage Include="System.ServiceProcess.ServiceController" />
-    <LibraryPackage Include="System.Text.Encoding.CodePages" />
-    <LibraryPackage Include="System.Threading.AccessControl" />
     <LibraryPackage Include="System.ServiceModel.Primitives" />
     <LibraryPackage Include="System.ServiceModel.Duplex" />
     <LibraryPackage Include="System.ServiceModel.Http" />
     <LibraryPackage Include="System.ServiceModel.NetTcp" />
     <LibraryPackage Include="System.ServiceModel.Security" />
-    <PrereleaseLibraryPackage Include="System.Data.Odbc" />
-    <PrereleaseLibraryPackage Include="System.Drawing.Common" />
-    <PrereleaseLibraryPackage Include="System.Diagnostics.EventLog" />
-    <PrereleaseLibraryPackage Include="System.Diagnostics.PerformanceCounter" />
-    <PrereleaseLibraryPackage Include="System.ServiceModel.Syndication" />
   </ItemGroup>
 
   <ItemGroup>
     <Dependency Include="@(LibraryPackage)" />
     <Dependency Include="@(PrereleaseLibraryPackage)" />
+
+    <!-- Include dependency to shims package -->
+    <Dependency Include="Microsoft.NETFramework.Compatibility.Shims">
+      <Version>$(CompatibilityShimsPackageVersion)</Version>
```

Comment:
Won't this need to include the prerelease version?

Reply:
oh nvm I think this is done automatically.

Reply:
Yes that is done automatically since the package is nor marked as stable.